File: PHP/car.php
<?php
class Car{
    public $id;
    public $license;
    public $driver;
    protected $passenger;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }
    
    public function printDataCar(){
        echo "license: $this->license, driver: {$this->driver->name}, document: {$this->driver->document}";

    }

    public function getpassenger() {
        return $this -> passenger;
    }


    public function setpassenger($passenger){
        if ($passenger == 4) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 4 pasajeros";
        }
    }
}

File: Java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car (String license, Account driver){
        this.license = license;
        this.driver = driver;
        // passenger = 3;
    }

    void printDataCar(){
        if (passenger != null) {
            System.out.println("License: " + license + " Driver: "+ driver.name + " Passengers: "+ passenger);
        }
    }

    public Integer getpassenger(){
        return passenger;
    }

    public void setpassenger(Integer passenger){
        if (passenger == 4){
            this.passenger = passenger;  
        }else {
            System.out.println("Asigna 4 passeengers");
        }  
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
    

}
